src/ResourceManager/ServiceBus/Commands.ServiceBus.Test/ScenarioTests/ServiceBusSubscriptionTests.ps1

```diff
@@ -26,6 +26,9 @@ function ServiceBusSubscriptionTests
 	$nameTopic = getAssetName "Topic-"
 	$subName = getAssetName "Subscription-"
 
+
+	# $resltNewSub = New-AzureRmServiceBusSubscription -ResourceGroupName "RGName-970" -Namespace "Namespace-786" -Topic "topic-3510" -Name "TestingSub1"
```

Comment:
@v-Ajnava please remove this if it's going to be commented out

Reply:
my bad, removed.